Posts/2021/04/NewTemporaryFoldersPowerShell

```diff
@@ -0,0 +1,89 @@
+---
+post_title: Borrowing a Built in PowerShell to create a Temporary Folder
+username: sean.kearney@microsoft.com
+Catagories: PowerShell, Function
+tags: Function,Fun trick,Existing Cmdlet,New Purpose
+Summary: Leveraging Built in Cmdlets in a New and interesting ways
+---
+
+**Q:** Hey question for you.  It seems silly and I know I could probably put something together with Get-Random.  But can you think of another way to create a temporary folder with a random in PowerShell? Ideally I'd like it to be in a user's own "Temporary Folder" is possible.
+
+**A:**  We sure can!  If Doctor Scripto was sitting here right now I'd see that little green haired shout out "Never fear, Scripto is here!"
+
+
+## New-TemporaryFile Cmdlet
+
+Within PowerShell there is a built in Cmdlet called “New-TemporaryFile”.  Running this Cmdlet simply creates a random 0 byte file in the $ENV:Temp folder in whichever platform you are working in.
```

Comment:
Within PowerShell there is a built-in Cmdlet called `New-TemporaryFile`.  Running this Cmdlet creates a random 0-byte file in the $ENV:Temp folder in whichever platform you are working.